Bottom: e51bd5e66eef5c09913b4b9c2ad58fb502870993
Top:    39c43bc1d966cdda6123b9999bd0389f6f7223f3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-05-16 20:50:17 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
